registry/auth/token/token.go

```diff
@@ -34,6 +34,7 @@ var (
 // ResourceActions stores allowed actions on a named and typed resource.
 type ResourceActions struct {
 	Type    string   `json:"type"`
+	Class   string   `json:"class"`
```

Comment:
Should we name this `Subtype` instead of `Class` to make it more consistent with the grammar? The term "class" so far is only used to describe the subtype of a repository, not applied to a resource.